addon/components/power-select-option/component.js

```diff
@@ -0,0 +1,22 @@
+import Component from '@ember/component';
+import layout from './template';
+import { computed } from '@ember/object';
+
+export default Component.extend({
+  tagName: 'li',
+  classNames: ['ember-power-select-option'],
+  layout,
+
+  iconName: computed('icon', function() {
```

Comment:
It's not more easy set `this.icon` as the rigth icon name ?

Reply:
you mean just return this.icon directly? the response i get has key names of list, mailing, stream, campaign. but the icons itself are monitor, search, inbox and workflow so it's not the exact same name.